### ISSUE TITLE
Cannot pre-compile templates using Ember RC3

### ISSUE THREAD

**Issue by MilkyWayJoe** [Myslik/csharp-ember-handlebars#15]
> Uncaught TypeError: Object function n(u,i,f,e){if("string"==typeof u&&(f=n,n=u,u=null),!u||-1!==n.indexOf("."))return i(u,n,f,e);var h,s,o=u[r],c=o&&o.descs[n];return c?c.set(u,n,f):(h="object"==typeof u&&!(n in u),h&&"function"==typeof u.setUnknownProperty?u.setUnknownProperty(n,f):o&&o.watching[n]>0?(s=t?o.values[n]:u[n],f!==s&&(Ember.propertyWillChange(u,n),t?void 0!==s||n in u?o.values[n]=f:Ember.defineProperty(u,n,null,f):u[n]=f,Ember.propertyDidChange(u,n))):u[n]=f),f} has no method 'indexOf' 
> base:4302
> 
> Uncaught TypeError: Object function n(t,i){if(""===i)return t;if(i||"string"!=typeof t||(i=t,t=null),!t||-1!==i.indexOf("."))return r(t,i);var r,u=t[f],o=u&&u.descs[i];return o?o.get(t,i):(n=e&&u&&u.watching[i]>0?u.values[i]:t[i],void 0!==n||"object"!=typeof t||i in t||"function"!=typeof t.unknownProperty?n:t.unknownProperty(i))} has no method 'create' base:8578

I still haven't found the cause of the issue

**Comment by Myslik** [comment]
I will try some debugging today after work.

**Comment by MilkyWayJoe** [comment]
I'm investigating it too. I first thought it was because of handlebars runtime, but it doesn't seem to be it

**Comment by MilkyWayJoe** [comment]
The issue is the way the .NET Bundler minifies Ember. I have created a static version of the sample app based on the scripts of a build (just saved the bundles as js files), having a separate bundle for Ember and another for Ember-Data. The static html didn't run, so I replaced the minified Ember and Ember-Data scripts from the version in the min files from their NuGet packages and the app ran fine along with the other files minified by the VS app build. 

I then changed the sample app so it doesn't bundle these scripts and changed their references to `script` tags in the `_Layout.cshtml` file and the app ran on build.

We have two temporary solutions: Not bundle Ember and Ember-Data and include them via script tags directly in the _Layout.cshtml file:

```
<script type="text/javascript" src="@Url.Content("~/scripts/ember-1.0.0-rc.3.min.js")"></script>
<script type="text/javascript" src="@Url.Content("~/scripts/ember-data.min.js")"></script>
```

Or we could write a temporary bundle transform implementation just for Ember & Ember-Data that simply returns the already minified files (from packages), keeping the Bundle from messing up with Ember during build.

Whichever option we take, we must add (more complete) documentation in a Readme file which must be included in the NuGet package. I will submit a PR for the first option later today or sometime tomorrow.
